hue/README.md

```diff
@@ -0,0 +1,49 @@
+# Cloudera Hue
+
+Deploys the Cloudera Hue server allowing data exploration on Hive and S3 buckets.
+
+Cloudera Hue is expected to be deployed along any HiveServer2 type of service. In Open Data Hub a [Spark SQL Thrift Server](../thriftserver) is used. Without Thrift Server deployment, Hue won't be able to fulfil any SQL queries. However it can still serve as an S3 browser.
+
+### Folders
+
+There is one main folder in the Hue component `hue` which contains the kustomize manifests.
+
+### Installation
+
+To install Hue add the following to the `kfctl` yaml file.
+
+```yaml
+- kustomizeConfig:
+    repoRef:
+      name: manifests
+      path: hue/hue
+  name: hue
+```
+
+### Overlays
+
+Hue component provides a single overlay.
+
+#### storage-class
+
+Customizes Hue's database to use a specific `StorageClass` for PVC, see `storage_class` parameter.
+
+### Parameters
+
+There are 4 parameters exposed vie KFDef.
+
+#### storage_class
+
+Name of the storage class to be used for PVC created by Hue's database. This requires `storage-class` **overlay** to be enabled as well to work.
+
+#### hue_secret_key
+
+Set session store secret key for Hue web server.
+
+#### s3_endpoint_url
```

Comment:
Can you add `s3_is_secure` to the parameters in the readme? It would be required for testing this with the unsecure ceph-nano-0 endpoint

Reply:
Oh.. I've removed that variable and re-add it. And I forgot to re-add the readme section. Just a sec. Appologies.

Reply:
@LaVLaS done